.clang-format

```diff
@@ -9,11 +9,11 @@ AlignEscapedNewlines: Right
 AlignOperands:   true
 AlignTrailingComments: true
 AllowAllParametersOfDeclarationOnNextLine: true
-AllowShortBlocksOnASingleLine: false
```

Comment:
**Review note**: This was changed, as `AllowShortBlocksOnASingleLine` is a `Never`/`Empty`/`Always` value.